cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/relational/RelationalTranformContext.java

```diff
@@ -66,4 +69,12 @@ public interface RelationalTranformContext extends FeatureFlagsProvider {
    *
    */
   void setOutputRelation(String portName, Relation outputDataSet);
+
+  /**
+   * Gets the list of languages that the relation transform context supports by default.
```

Comment:
Please phrase is as next: "Provides a list of capabilities that plugins must use to locate an ExpressionFactory to parse user-entered expressions." I am wondering if we should make it more explicit by naming e.g. `getUserExpressionCapabilitySelector` or similar. @fernst WDYT?